docs/website/root/manual/developer-docs/nodes/mithril-client.md

```diff
@@ -358,20 +362,20 @@ Here is a list of the available parameters:
 | `artifact_hash` | `--artifact-hash` | - | - | Hash of the Mithril stake distribution artifact or `latest` for the latest artifact | - | - | :heavy_check_mark: |
 | `download_dir` | `--download-dir` | - | - | Directory where the Mithril stake distribution will be downloaded | . | - | - |
 
-`cardano-transaction --unstable sets show` command:
+`cardano-transaction commitment show` command:
```

Comment:
It looks like the `--unstable` parameter is missing here and below

Reply:
It's quite complicated here, in fact the `--unstable` parameter is not a part of the `cardano-transaction` command, it belongs to the main command.
So if we really want to put it here it would be ... at the start: `--unstable cardano-transaction commitment show` and I think that it makes this part of the documentation harder to read.

Reply:
Yes you're right, it belongs to the main command :+1: